README.md

```diff
@@ -13,7 +13,7 @@ This client presents an API access to these services and data.
 ## Installation
 
 ```bash
-$ pip install 'ctadata>=0.2.6'
+pip install 'ctadata>=0.2.6'
```

Comment:
```suggestion
$ pip install 'ctadata>=0.2.6'
```

Reply:
Ok, I can reverse this change if you want. I removed this because it's clear that we need to run this line and because when using the copy button, we do not have to remove this extra character manually. It's mainly a question of simplicity for me.

Reply:
It's commonly done in one way or another, there are arguments for and against. 
In ambiguous cases, like this one, I think it's best to give the code author some leeway, not changing parts of the code to own preference in unrelated PR.

So yes, let's change back. We can discuss separately if it's best to do something different which will allow both distinct presentation, copying, and other features if any.